packages/core/src/components/field-label/field-label.tsx

```diff
@@ -183,7 +183,7 @@ export class FormFieldLabel implements IxComponent {
           ref={this.labelRef}
         >
           <ix-typography
-            color={this.isInvalid ? 'alarm' : 'soft'}
+            textColor={this.isInvalid ? 'alarm' : 'soft'}
```

Comment:
```suggestion
            text-color={this.isInvalid ? 'alarm' : 'soft'}
```

Reply:
```suggestion
            textColor={this.isInvalid ? 'alarm' : 'soft'}
```
textColor should be okay in here, shouldn't it?

Reply:
textColor should be okay in here, shouldn't it?